scoresheets/StoringGroceries.tex

```diff
@@ -3,7 +3,7 @@
 
 \begin{scorelist}
 	\scoreheading{Main Goal}
-	\scoreitem{500}{Move 5 objects next to their peers in the shelf}
+	\scoreitem[5]{100}{Move an objects next to their peers in the shelf}
```

Comment:
Careful!

```\scoreitem{500}{Move 5``` means that team won't score unless 5 objects are moved.
```\scoreitem[5]{100}{Move an``` means one object is enough to start scoring bonus points.

I agree on the 2nd one. HOWEVER: *move an objects*?

Reply:
Yeah should be "an object"
I think second option is more in line with scoring in clean up task which seems similar in some ways.

Reply:
:+1